vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -630,4 +630,11 @@ file_management_s3_url: "http://acm-arkcase:9000"
 file_management_s3_access_key: "minioadmin"
 file_management_s3_secret_key: "minioadmin"
 file_management_s3_buckets:
-  - arkcase-portal
+  - config_name: arkcaseBucket
+    bucket_name: arkcase-portal
```

Comment:
In all facts files, this bucket_name is without "-".

Reply:
This is one object in the list entry with two properties. Similar to ldap_groups fact.